### DISCUSSION TITLE
is this possible to do to Firemote cards?

### DISCUSSION THREAD

**Discussion by MK00000000000000000000** [PRProd/HA-Firemote#449] [General]
Ha-Firemote is awesome. Really cudos!

Just wondering, and fyi I am not a dashboard guru yet... but is it possible to put 5 configured firemote cards in a single card, with swipe to the right functionality to go through them each, one at a time? Not sure if it's possible with cards like this. Thanks!

**Comment by PRProd** [comment]
Greetings @MK00000000000000000000!!!  ~

Thanks for your kind words, they truly mean a lot to me.  I'm just getting back from a long vacation so I'm trying to dig through all of my work, personal, and Firemote messages today.  There's a lot to do.  😄 

I have not personally used any sort of "swipe" card to contain other cards before, but I would imagine that one or more of those things exist somewhere in HACS.  What I can tell you is that you _can_ have as many Firemote cards on your dashboard as you'd like.  So, if you find a suitable "swipe" card style container to put them in, you should have no problems! 👍🏻 

Also as a fun idea, you might want to check out something that user @neilimixamo posted last year in the Firemote [Let's See Your Dashboards!](https://github.com/PRProd/HA-Firemote/discussions/129) discussion.  The post contains a [video](https://github.com/PRProd/HA-Firemote/discussions/129#discussioncomment-6265750) and the [dashboard YAML code](https://github.com/PRProd/HA-Firemote/discussions/129#discussioncomment-6266031) used to create a unique solution for a mobile dashboard that allows for a Firemote card to pop-up when needed, and hide when it's not.

I hope this helps!  Enjoy your dashboard journey!

-Doug
